README.md

```diff
@@ -21,14 +21,56 @@ Once you have implemented the `fm` program (and tested it to the best of your ab
 
 *What preprocessing data do you store in files, and how?*
 
+We store the preprocessed data in a .txt file with \n between each data point. For multiple sequnences in the same fasta file, one is stored after the other. We store the following:
+* The sequence header. This is needed for the output.
+* n, the length of the sequence. Can be deduced from the other data, but make sense to save, since we can then malloc with it. 
+* σ, alphabetsize. 
+* bwt of the converted sequence (where the chars are ints 0...σ-1). With commas between each char. We could same O and C, but they would take more space. 
+* SA. Comma seperated
+* The alphabet. No seperation chars, since each symbol is a unique char. 
```

Comment:
Ok. You could also just dump all the arrays (and serialize the O-table as one long array) directly into a binary file. It is a lot faster to dump and load that way than to parse a textual file. But it probably doesn't matter so much in this application.